.github/workflows/filename_workflow.yml

```diff
@@ -0,0 +1,45 @@
+name: aws-api-filename-test
+on:
+  workflow_dispatch: # allow workflow to be manually triggered
+  repository_dispatch:
+    types: [trigger-profiler]
```

Comment:
No need to do: 
```
types: [trigger-profiler]
```
if we don't need to run it per dev deployment. Maybe this can just be triggered manually?

Reply:
Yep I'll leave it just with the dispatch